<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetPointCloudProDialog</class>
 <widget class="QDialog" name="SetPointCloudProDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>20</y>
     <width>387</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Red Channel:</string>
      </property>
      <property name="buddy">
       <cstring>rSlider</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QSlider" name="rSlider">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLCDNumber" name="RlcdNumber">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Green Channel:</string>
      </property>
      <property name="buddy">
       <cstring>gSlider</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QSlider" name="gSlider">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLCDNumber" name="GlcdNumber"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Blue Channel:</string>
      </property>
      <property name="buddy">
       <cstring>bSlider</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QSlider" name="bSlider">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLCDNumber" name="BlcdNumber"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>PointSize:</string>
      </property>
      <property name="buddy">
       <cstring>sizeSlider</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QSlider" name="sizeSlider">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>15</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLCDNumber" name="PointSizelcdNumber"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>PointCloudID:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="pointCloudID"/>
    </item>
    <item row="4" column="2" colspan="2">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SetPointCloudProDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SetPointCloudProDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RlcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>GlcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BlcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sizeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PointSizelcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
